govc/pool/destroy.go

```diff
@@ -39,7 +40,8 @@ func (cmd *destroy) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.DatacenterFlag, ctx = flags.NewDatacenterFlag(ctx)
 	cmd.DatacenterFlag.Register(ctx, f)
 
-	f.BoolVar(&cmd.recursive, "r", false, "Remove all child resource pools recursively")
+	f.BoolVar(&cmd.recursive, "r", false, "Remove all child resource pools recursively - DEPRECATED")
+	f.BoolVar(&cmd.children, "children", false, "Remove all chilren pools")
```

Comment:
Typo in the usage.  Yes, lets remove the -r flag, very unlikely it is being used.